uberon_edit.obo

```diff
@@ -98409,7 +98409,7 @@ xref: FBbt:00007060
 xref: FMA:83115
 is_a: UBERON:0000477 {source="FMA"} ! anatomical cluster
 relationship: composed_primarily_of GO:0005575 ! cellular_component
-relationship: has_component GO:0044464 {minCardinality="2"} ! cell part
+relationship: has_component CL:0000000 {minCardinality="2"} ! cell
```

Comment:
this is incorrect for same reasons as the above

I think the value of the axiom is even lower, so remove